core/src/main/java/de/danoeh/antennapod/core/util/ChapterUtils.java

```diff
@@ -123,6 +135,23 @@ private static CountingInputStream openStream(Playable playable, Context context
         }
     }
 
+    public static List<Chapter> loadChaptersFromUrl(String url) {
+        List<Chapter> chapters;
+        Request request = new Request.Builder().url(url).build();
+        try {
+            Response response = AntennapodHttpClient.getHttpClient().newCall(request).execute();
+            if (response.body() == null) {
+                return null;
+            }
+            chapters = PodcastIndexChapter.parseChapters(response.body().toString());
+            return chapters;
```

Comment:
You can save a line of code here ;)

```suggestion
            return PodcastIndexChapter.parseChapters(response.body().toString());
```